ops/services/okta-app/_var.tf

```diff
@@ -28,6 +28,40 @@ variable "redirect_urls" {
   ]
 }
 
+variable "trusted_origin_preview_urls" {
+  description = "Trusted origin listing for the Okta Preview environment. Data format is [Name, URL, [Scopes]]"
+  default = [
```

Comment:
Should we also include `localhost:3000` in here as well?

Reply:
Good question! This is an exact copy of what currently exists in the preview environment, at least from what I can see. That would be why there is nothing else in this list.

Reply:
AFAIK we were hesitant to include any non-CDC-managed domains in the Okta allowlist, which is why `localhost` isn't included.

Reply:
Let's leave `localhost` origins out just to be sure.

Reply:
Done!